newsletter/models.py

```diff
@@ -1,18 +1,52 @@
 from django.db import models
 from django.utils import timezone
+import uuid
+
+# Custom Manager for active subscribers
+class ActiveSubscriberManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(is_active=True)
```

Comment:
I think instead of `is_active` field, we must add a `disactivated_at` field. Use `timestamps`